rollup/withdrawtrie/withdraw_trie.go

```diff
@@ -0,0 +1,18 @@
+package withdrawtrie
+
+import (
+	"github.com/scroll-tech/go-ethereum/common"
+	"github.com/scroll-tech/go-ethereum/rollup/rcfg"
+)
+
+// StateDB represents the StateDB interface
+// required to get withdraw trie root
+type StateDB interface {
+	GetState(common.Address, common.Hash) common.Hash
+}
+
+// ReadWTRSlot reads WithdrawTrieRoot slot in L2MessageQueue predeploy, i.e., `messageRoot`
+// in scroll-tech/scroll/contracts/src/libraries/common/AppendOnlyMerkleTree.sol
```

Comment:
```suggestion
// in contracts/src/libraries/common/AppendOnlyMerkleTree.sol
```

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/217/commits/40668d88ae01833e0eadd5e183aa6875598f410b